dali/base/dafdesc.cpp

```diff
@@ -1340,17 +1352,31 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
             if (_interleaved)
                 *_interleaved = interleaved;
         }
-    }
 
+        const char * historyStr = "History";
```

Comment:
OK, What kind of history? In a byte or int variable? What kind of versioning, sequence number, bit field?